kibbeh/public/locales/sv/translation.json

```
@@ -274,6 +287,8 @@
       "galacticDoge": "Galaktiskdoge",
       "spottedLife": "Planet med liv upptäckt"
     },
-    "feed": { "yourFeed": "Your Feed" }
+    "feed": {
+      "yourFeed": "Ditt flöde"
+    }
   }
 }
```

New line is missing:
```suggestion
}

```

---

Didn't I fix this?

---

It did not appear to be resolved. I just realized I can commit this myself so I'll do that then merge.